modules/rasBidAdapter.js

```diff
@@ -129,6 +129,36 @@ const getGdprParams = (bidderRequest) => {
   return queryString;
 };
 
+const parseAuctionConfigs = (serverResponse, bidRequest) => {
+  if (isEmpty(serverResponse) || isEmpty(bidRequest)) {
+    return null;
```

Comment:
why not append fledge if it is empty?

Reply:
In the first iteration of implementation we were planning to return the configs in the serverResponse, but as you pointed out now there is no reason to do that. I'll correct the code.